bandit/blacklists/calls.py

```diff
@@ -569,4 +569,65 @@ def gen_blacklist():
         'attacks. Consider using tmpfile() instead.'
         ))
 
+    # updated rulesets starts here
+
+    sets.append(utils.build_conf_dict(
+        'yaml_load', 'B326',
+        ['yaml.load'],
+        """
+        Use of this deprecated deserialize method leads to
+        remote code execution. See CVE-2017-18342.
+        """
+        'Use yaml.safe_load instead.'
+        ))
+
+    sets.append(utils.build_conf_dict(
+        'response_splitting', 'B327',
+        [
+            'urllib2.Request.add_header',
+            'urllib.URLopener.addheader',
+            'urllib.URLopener.addheaders'
+        ],
+        'Python Urllib and Urllib2 are vulnerable to CRLF injection.'
+        'See https://bugs.python.org/issue36276'
+        ))
+
+    sets.append(utils.build_conf_dict(
+        'Base64_encoding_used', 'B328',
```

Comment:
There are some very good use cases to use base64. In fact, some protocols require it. So there's a very low confidence this check would find a security issue. I'd rather not introduce many more false positives.